manifests/agent.pp

```diff
@@ -88,22 +88,17 @@
   Optional[Hash[String[1],String]] $annotations = undef,
   Optional[Hash[String[1],String]] $labels = undef,
   String[1] $namespace = 'default',
-  Optional[Array[String[1]]] $redact = undef,
+  Array[String[1]] $redact = ['password','passwd','pass','api_key','api_token','access_key','secret_key','private_key','secret'],
```

Comment:
are these the default values that sensu ships with?

Reply:
Those are the defaults from Sensu.